### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by fregante**

### CODE REVIEW

**Comment by kidonng** [comment]
You might want to do https://github.com/vadimdemedes/dom-chef/pull/80#discussion_r791482402

**Comment by fregante** [comment]
Good point, it makes sense in this lint PR

**Comment by fregante** [comment]
@kidonng good to go? Last chance before I release 😃

File: package.json [hunk 1]
```diff
@@ -81,9 +74,9 @@
 		"ava": "^4.1.0",
 		"eslint-plugin-react": "^7.29.4",
 		"jsdom": "^19.0.0",
-		"npm-run-all": "^4.1.5",
+		"rollup": "^2.70.1",
```
Comment (kidonng) [inline package.json:77]: No longer necessary

```suggestion
```

File: package.json [hunk 2]
```diff
@@ -15,7 +15,6 @@
 		"typescript"
 	],
 	"repository": "vadimdemedes/dom-chef",
-	"funding": "https://github.com/sponsors/fregante",
```
[the remark this answers is not in the document]
Comment (fregante) [reply package.json]: C-658VsXoAo3ovC

File: package.json (hunk 1 above)
Comment (fregante) [reply package.json]: b0db501b-785d-485c-b9c2-b99510d97f40_text

**Comment by fregante** [comment]
8c1c57a5ee93c36585c151ecfd6927ea

**Pull request merged**